mailbox/api/src/test/java/org/apache/james/mailbox/MailboxManagerTest.java

```diff
@@ -999,7 +998,7 @@ void searchShouldNotReturnResultsFromOtherNamespaces() throws Exception {
             session = mailboxManager.createSystemSession(USER_1);
             mailboxManager.createMailbox(new MailboxPath("other_namespace", USER_1, "Other"), session);
             mailboxManager.createMailbox(MailboxPath.inbox(session), session);
-            List<MailboxMetaData> metaDatas = mailboxManager.search(
+            List<org.apache.james.mailbox.model.MailboxMetaData> metaDatas = mailboxManager.search(
```

Comment:
Is this to force the differenciation with `MessageManager.MailboxMetaData`? Is it necessary?